src/backend/cli/constants.py

```diff
@@ -0,0 +1,123 @@
+from enum import StrEnum
```

Comment:
Couple refactorings:

Split up the code into
- constants
- prompts
- utils
- setters

Reply:
Most of the new code is in setters (setting YAML config)

Reply:
Nice!